server/src/immich/controllers/person.controller.ts

```diff
@@ -52,4 +53,9 @@ export class PersonController {
   getPersonAssets(@AuthUser() authUser: AuthUserDto, @Param() { id }: UUIDParamDto): Promise<AssetResponseDto[]> {
     return this.service.getAssets(authUser, id);
   }
+
+  @Post('merge')
```

Comment:
I think it should be :id/merge. The client should specify who the "original" person is.